### PR TITLE
creates a stack that creates a vpc

### PR DESCRIPTION
**Pull request by hkothari128**
image

image

image

image

### CODE REVIEW

File: CDK/lib/cdk-stack.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          
+        },
+        {
+          subnetType:ec2.SubnetType.ISOLATED,
+          name: 'isolatedSN2',
+          cidrMask:24,
+        }
+      ]
+
+    })
```
Comment (hkothari128) [inline CDK/lib/cdk-stack.ts]: will need a ; here if we are following some stricter linting rules.

Comment (husyn) [reply CDK/lib/cdk-stack.ts]: can you update the PR then?

**Comment by husyn** [comment]
Can we delete the test files? We don't need those for the time being.

**Review by husyn — APPROVED** [review]
LGTM (Looks Good To Me)

**Pull request merged**